contents/docs/integrate/proxy.mdx

```diff
@@ -56,5 +56,11 @@ Community Slack user CJ Enright shared their AWS CDK code for creating this reve
 
 You can view that here: https://gist.github.com/pauldambra/9af02d9ea42ffafcfc7c01dc38039958
 
-### Summary
-That's it! A few resources here will go a long way. Reach out if you hit any snags!
+## Cloudflare Proxy
```

Comment:
Does CloudFlare update the `Host` header or allow setting it? If not then this will already be an issue with `eu.posthog.com` and will soon be an issue with US as well.